src/components/ProductDetailAddToCart/ProductDetailAddToCart.js

```diff
@@ -9,8 +9,39 @@ import Typography from "@material-ui/core/Typography";
 import Minus from "mdi-material-ui/Minus";
 import Plus from "mdi-material-ui/Plus";
 import { inject, observer } from "mobx-react";
+import CartPopover from "components/CartPopover";
 import Divider from "components/Divider";
 
+// This is a temporary cartItem object to be used for testing
+// pending the GraphQL endpoint being hooked up
+// Remove the code between these comments when live data is available
+const cartItem = {
+  _id: "abcdefghijklmnop",
+  attributes: [
+    {
+      label: "Color",
+      value: "Red"
+    },
+    {
+      label: "Season",
+      value: "Summer"
+    }
+  ],
+  currentQuantity: 10,
+  imageUrl: "http://localhost:3000/assets/files/Media/nbYKLrZST5DSF87md/thumbnail/chuttersnap-265339-unsplash.png",
```

Comment:
Can we make this temp image come from placehold.it? I've been using `http://placehold.it/100` for the comp-library